src/libraries/System.IO.FileSystem/tests/LargeFileTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO.Tests;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.FileSystem.Tests
+{
+    [OuterLoop]
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/45954", TestPlatforms.Browser)]
+    [Collection("NoParallelTests")] // don't create multiple large files at the same time
```

Comment:
nit: we can use `nameof` to reduce the chance for typo

```suggestion
    [Collection(nameof(NoParallelTests))] // don't create multiple large files at the same time
```